### ISSUE TITLE
Issue when creating ServiceNow Crossplane Provider code

### ISSUE THREAD

**Issue by gaurav218** [crossplane/terrajet#258]
This is not a bug due to Terra Jet provider but I am facing an issue while creating ServiceNow provider code using Terra Jet provider template.  When I run make generate it displays below error -

14:18:25 [ .. ] verify dependencies have expected content
all modules verified
14:18:25 [ OK ] go modules dependencies verified
14:18:25 [ .. ] generating provider schema for integrations/servicenow 0.9.5
make[1]: *** [Makefile:90: config/schema.json] Error 1
make: *** [build/makelib/common.mk:418: generate] Error 2

My guess is this could be due to the fact that Terraform ServiceNow provider is a custom provider and thats why terra jet is unable to create a proper schema.json. I am only trying to create code for "application" managed resource in ServiceNow.

Can anyone suggest where I am doing wrong here -

Pl. find my makefile export section snippet.
export TERRAFORM_VERSION := 1.1.6

export TERRAFORM_PROVIDER_SOURCE := integrations/servicenow
export TERRAFORM_PROVIDER_VERSION := 0.9.5
export TERRAFORM_PROVIDER_DOWNLOAD_NAME := terraform-provider-servicenow
export TERRAFORM_PROVIDER_DOWNLOAD_URL_PREFIX := https://github.com/tylerhatton/terraform-provider-servicenow/releases/v0.9.5

schema.json format -

{"format_version":"1.0","provider_schemas":{"registry.terraform.io/hashicorp/null":{"provider":{"version":0,"block":{"description_kind":"plain"}},"resource_schemas":{"null_resource":{"version":0,"block":{"attributes":{"id":{"type":"string","description":"This is set to a random value at create time.","description_kind":"markdown","computed":true},"triggers":{"type":["map","string"],"description":"A map of arbitrary strings that, when changed, will force the null resource to be replaced, re-running any associated provisioners.","description_kind":"markdown","optional":true}},"description":"The `null_resource` resource implements the standard resource lifecycle but takes no further action.\n\nThe `triggers` argument allows specifying an arbitrary set of values that, when changed, will cause the resource to be replaced.","description_kind":"markdown"}}},"data_source_schemas":{"null_data_source":{"version":0,"block":{"attributes":{"has_computed_default":{"type":"string","description":"If set, its literal value will be stored and returned. If not, its value defaults to `\"default\"`. This argument exists primarily for testing and has little practical use.","description_kind":"markdown","optional":true,"computed":true},"id":{"type":"string","description":"This attribute is only present for some legacy compatibility issues and should not be used. It will be removed in a future version.","description_kind":"markdown","deprecated":true,"computed":true},"inputs":{"type":["map","string"],"description":"A map of arbitrary strings that is copied into the `outputs` attribute, and accessible directly for interpolation.","description_kind":"markdown","optional":true},"outputs":{"type":["map","string"],"description":"After the data source is \"read\", a copy of the `inputs` map.","description_kind":"markdown","computed":true},"random":{"type":"string","description":"A random value. This is primarily for testing and has little practical use; prefer the [hashicorp/random provider](https://registry.terraform.io/providers/hashicorp/random) for more practical random number use-cases.","description_kind":"markdown","computed":true}},"description":"The `null_data_source` data source implements the standard data source lifecycle but does not\ninteract with any external APIs.\n\nHistorically, the `null_data_source` was typically used to construct intermediate values to re-use elsewhere in configuration. The\nsame can now be achieved using [locals](https://www.terraform.io/docs/language/values/locals.html).\n","description_kind":"markdown","deprecated":true}}}}}}

**Comment by turkenh** [comment]
@gaurav218 could you check if you have any details in the following file `.work/terraform/terraform-logs.txt`? 

https://github.com/crossplane-contrib/provider-jet-template/blob/main/Makefile#L91

**Comment by gaurav218** [comment]
Thnx for the response, the issue was with wrong source value.

I did ran "make generate" and now getting below logs in /terraform-logs.txt file-

Initializing the backend...

Initializing provider plugins...
- Finding tylerhatton/servicenow versions matching "0.9.5"...
- Installing tylerhatton/servicenow v0.9.5...
- Installed tylerhatton/servicenow v0.9.5 (self-signed, key ID 4B1A29E261D0350A)

Partner and community providers are signed by their developers.
If you'd like to know more about provider signing, you can read about it here:
https://www.terraform.io/docs/cli/plugins/signing.html

Terraform has created a lock file .terraform.lock.hcl to record the provider
selections it made above. Include this file in your version control repository
so that Terraform can guarantee to make the same selections by default when
you run "terraform init" in the future.

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.


But I received following error in cmd -

build command-line-arguments: cannot load embed: malformed module path "embed": missing dot in first path element
apis/generate.go:33: running "go": exit status 1
generate: open /home/azureuser/provider-jet-template/apis/v1alpha1/zz_generated.deepcopy.go: no such file or directory
11:10:37 [FAIL]
make[1]: *** [build/makelib/golang.mk:271: go.generate] Error 1
make: *** [build/makelib/common.mk:419: generate] Error 2
root@ubuntuvm2:/home/azureuser/provider-jet-template# ls

**Comment by gaurav218** [comment]
Guess I need to upgrade go version to 1.16 or above

**Comment by gaurav218** [comment]
@turkenh, thnx for your support. Is it possible to have 15 minutes teams session as I am getting some error related to github username not provided. This provider code is related to servicenow so do I need to pass github username and ped in providerconfig.yaml file??

**Comment by gaurav218** [comment]
@turkenh any pointers here on above issue. The target state is to manage this multicloud environment using Crossplane. We need custom servicenow provider to create managed resources for apps teams. Can you suggest what are the exact steps to achieve it.

**Comment by turkenh** [comment]
@gaurav218 it is a bit hard to follow up here, any chance that you can share your latest state with a repo/PR so that I can check/reproduce the issue on my side?
